packages/edit-site/src/components/page-templates-template-parts/index.js

```diff
@@ -262,6 +271,12 @@ export default function PageTemplatesTemplateParts( { postType } ) {
 		} ) );
 	}, [ records ] );
 
+	const sources = useMemo( () => ( {
+		theme: __( 'Theme' ),
+		plugin: __( 'Plugin' ),
+		user: __( 'User' ),
```

Comment:
In the sidebar of the site editor, it says "custom" and here it says "user". I believe we should use the same label no?

Reply:
Good question, that's one thing I'd like to get feedback on:

* In the sidebar, "Custom" feels more intuitive to me. In WP jargon we typically see "custom templates", and rarely "user templates". We could switch to "User", but that feels confusing.
* However, in the filter, _Source is Custom_ felt like a weird rendering, so I kept it true to the internal naming: _Source is User_.

What should we do? Both directions seem weird to me.

Reply:
I think "User" templates in the sidebar is fine by me too. But would love feedback from @jameskoster

Reply:
Thanks for the ping. Given this filter shows any user-generated templates, I agree it's fine for the label in the sidebar to be 'User'.

However, it would be good to have a 'Custom' view as well, which should show any `$custom` templates according to the [template hierarchy](https://developer.wordpress.org/themes/basics/template-hierarchy/#visual-overview). This would match the 'Custom' section in the current template list. I acknowledge there's no way to filter in this manner yet, so I guess it'll need to happen separately. Perhaps we should add a 'type' taxonomy?

Reply:
I mean if the goal is to have separation according to "custom", "non custom" templates, that's possible to, it's just a different field. But the question is also what to do for "plugin templates" because these also can be custom or not. 

So I guess before moving on here, let's get some clarity about what we want for 6.5 from a design perspective.

Reply:
If a plugin (or theme) template is custom then it should appear in the "Custom" view, just as those templates appear currently under the "Custom" heading in the Templates sidebar:

<img width="350" alt="Screenshot 2024-02-12 at 21 07 26" src="https://github.com/WordPress/gutenberg/assets/846565/7a342bd1-795e-494b-9d18-62d90a806ba2">


For 6.5, at the least, I would suggest replicating the current list, IE a view for custom templates, and a view for each plugin that adds templates. Source = user is a nice-to-have, but since it doesn't exist already not essential.